homework_part_2/app.js

```diff
@@ -0,0 +1,77 @@
+//Part 1
+//1. Concatenation consists of appending one string to the end
+//of another using the "+" operator; interpolation requires
+//the evaluation of a variable into its corresponding value
+//Concat example: "Hello " + "World"
+//Interp example: const x = "Hello "
+//console.log(`${x} World`)
+
+//2. DRY stands for Don't Repeat Yourself; it allows for more
+//efficient coding. Tools such as switch and loops help us
+//write DRY code
+
+//3. Declaring a variable creates the variable; assigning a value
+//simply sets its value. Defining a variable consists of declaring
+//it and assigning its value simultaneously.
+
+//4. Use const when you don't want a variable to be updated;
+//use let when you do
+
+//5. A parent directory is a directory that contains
+//sub- or children directories
+
+//6. man
+
+//7. Tab completion allows you to click tab to autocomplete
+//directory or file names, thus saving much time
+
+//Part 2
+//1. <
+//2. >
+//3. ==
+//4. <, <
+//5. ==, !=
```

Comment:
`a == a != d` is not syntax that makes sense in JS.  take another look at this and the previous question and try something less arcane